critiquebrainz/data/dump_manager.py

```diff
@@ -288,17 +293,16 @@ def create_base_archive(*, location, meta_files_dir=None):
         # Dumping tables
         base_archive_tables_dir = os.path.join(base_archive_dir, 'cbdump')
         create_path(base_archive_tables_dir)
-        with db.engine.connect() as connection:
-            with connection.begin() as transaction:
-                cursor = connection.connection.cursor()
-                try:
-                    with open(os.path.join(base_archive_tables_dir, 'user_sanitised'), 'w') as f:
-                        cursor.copy_to(f, '"user"', columns=('id', 'created', 'display_name', 'musicbrainz_id'))
-                    with open(os.path.join(base_archive_tables_dir, 'license'), 'w') as f:
-                        cursor.copy_to(f, 'license', columns=_TABLES["license"])
-                except Exception:
-                    print('Error while copying tables during creation of the base archive.')
-                    transaction.rollback()
+
+        cursor = connection.connection.cursor()
+        try:
+            with open(os.path.join(base_archive_tables_dir, 'user_sanitised'), 'w') as f:
+                cursor.copy_to(f, '"user"', columns=('id', 'created', 'display_name', 'musicbrainz_id'))
+            with open(os.path.join(base_archive_tables_dir, 'license'), 'w') as f:
+                cursor.copy_to(f, 'license', columns=_TABLES["license"])
+        except Exception as e:
+            print('Error "{}" occurred while copying tables during creation of the base archive!'.format(e))
+            raise Exception
```

Comment:
if you do just `raise` here, it would raise the specific exception that was raised, which would be more helpful for debugging.

```suggestion
            raise 
```